output.tf

```diff
@@ -1,27 +1,3 @@
-output "advisor_recommendations" {
-  value = data.azurerm_advisor_recommendations.advisor.recommendations
-}
-
-output "aks_login" {
-  value = "az aks get-credentials --name ${module.kubernetes.name} --resource-group ${module.resource_group.name} --admin"
-}
-
-output "resource_group_name" {
-  value = "${module.resource_group.name}"
-}
-
-output "stop_cluster_cmd" {
-  value = "az aks stop --name ${module.kubernetes.name} --resource-group ${module.resource_group.name}"
-}
-
-output "start_cluster_cmd" {
-  value = "az aks start --name ${module.kubernetes.name} --resource-group ${module.resource_group.name}"
-}
-
-output "azure_region" {
-  value = module.resource_group.location
-}
-
-output "subscription_id" {
-  value = data.azurerm_subscription.current.subscription_id
-}
+output "eclwatchip" {
```

Comment:
Adding an output is OK for this PR, but there was no reason to remove the other outputs.  Please restore them.

Reply:
> Adding an output is OK for this PR, but there was no reason to remove the other outputs. Please restore them.

I stored all the outputs in pull request #5.